.changeset/sour-moons-rule.md

```diff
@@ -0,0 +1,5 @@
+---
+'@coinbase/onchainkit': patch
+---
+
+feat: Add isWalletASmartWallet utility. By @cpcramer
```

Comment:
You want to be more expressive here...

```
- **feat**: added the `isWalletASmartWallet` utility which helps ~~~~ . By @cpcramer #420
```

So give enough context that folks reading understand what's going on.